templates/reactor_config_map.yml

```diff
@@ -0,0 +1,221 @@
+apiVersion: v1
+data:
+  config.yaml: |
+    ---
+    allow_multiple_remote_sources: true
```

Comment:
remove all !! it is suppose to be just template :)

Reply:
and I'd have just simple template where you fill in tho whole config from group var like we have now

Reply:
Moved and changed